ui/src/sd-admin/studyAdmin.tsx

```diff
@@ -0,0 +1,497 @@
+import Autocomplete from "@mui/material/Autocomplete";
+import Backdrop from "@mui/material/Backdrop";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import CircularProgress from "@mui/material/CircularProgress";
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import TextField from "@mui/material/TextField";
+import { DataGrid, GridColDef } from "@mui/x-data-grid";
+import { useEffect, useState } from "react";
+import { useAdminSource } from "sd-admin/source";
+import { CreateStudyRequest, StudyV2, UpdateStudyRequest } from "tanagra-api";
+
+const columns = (
+  filterFn: (name: string, value: string) => void
+): GridColDef[] => [
+  {
+    field: "displayName",
+    headerName: "Study Name",
+    sortable: true,
+    disableColumnMenu: true,
+    flex: 1,
+    renderHeader: () => (
+      <div style={{ lineHeight: "1.5rem" }}>
+        <div>Study Name</div>
+        <div>
+          <TextField
+            sx={{
+              "label+.css-apw9r9-MuiInputBase-root-MuiInput-root": {
```

Comment:
I've found the MUI customization options fairly confusing but I think you want something like: `& .MuiInputBase-root": ...` and `& .MuiInputLabel-root": ...`. Or perhaps without the space between the `&` and `.`. I not sure the generated style names are guaranteed to by stable or something like that.

The documentation for customizing part of the MUI controls is here: https://www.figma.com/file/Dvq2jerBXPxACCvAFLYVSJ/Tanagra?node-id=101%3A303&t=Zywx8cTzczl4dMWv-0

Reply:
Yeah, I think I copied those values from the Chrome inspector but that makes sense that they would change. Both your suggestions seem to work, thanks.